distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc

```diff
@@ -27,4 +27,5 @@ include::modules/distr-tracing-architecture.adoc[leveloffset=+1]
 [id="additional-resources_distributed-tracing-architecture"]
 == Additional resources
 
-* xref:../../otel/otel-release-notes.adoc[{OTELName}]
+// xreffing to the installation page until further notice because OTEL content is currently planned for internal restructuring across pages that is likely to result in renamed page files
+* xref:../../otel/otel-installing.adoc[{OTELName}]
```

Comment:
This needs an anchor ID in the xref per [guidelines]([distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc](https://github.com/openshift/openshift-docs/pull/72079/files#diff-6ec699242de42a69712e13ee6cf9f4a288f58b7d1acedad41cbc30b89063db30)).

Reply:
Thank you. I've fixed this.